test/GitHub.InlineReviews.UnitTests/ViewModels/NewInlineCommentThreadViewModelTests.cs

```diff
@@ -10,6 +10,11 @@ namespace GitHub.InlineReviews.UnitTests.ViewModels
 {
     public class NewInlineCommentThreadViewModelTests
     {
+        public NewInlineCommentThreadViewModelTests()
+        {
+            Splat.ModeDetector.Current.SetInUnitTestRunner(true);
```

Comment:
We should probably move this into an assembly wide setup for the whole test assembly. This is possible with NUnit in a way it wasn't with xUnit.

For example we could move it into `AssemblyInfo.cs` like this:
https://github.com/github/VisualStudio/blob/ebb900e8ef58fd25af32d69421624bac5fe52c0b/test/GitHub.App.UnitTests/Properties/AssemblyInfo.cs#L45

Otherwise it's too easy to forget and end up with hanging or not depending on the order they're executed. 😨